include/model/CAnomalyDetectorModelConfig.h

```diff
@@ -162,6 +162,10 @@ class MODEL_EXPORT CAnomalyDetectorModelConfig {
     //! The default maximum time to test for a change point in a time series.
     static const core_t::TTime DEFAULT_MAXIMUM_TIME_TO_TEST_FOR_CHANGE;
 
+    //! The default number of time buckets used to generate multibucket features
+    //! for anomaly detection.
+    static const std::size_t MULTIBUCKET_FEATURE_WINDOW_LENGTH;
```

Comment:
nit: `MULTIBUCKET_FEATURE_WINDOW_LENGTH` in other places it is `MULTIBUCKET_FEATURES_WINDOW_LENGTH`, feature - > features